types/chai/index.d.ts

```diff
@@ -782,7 +782,7 @@ declare namespace Chai {
          * @param value    Actual value
          * @param message   Message to display on error.
          */
-        isFinite<T>(value: T, message?: string): void;
+        isFinite<T>(value: T, message?: string): asserts value is (T extends number ? T : never);
```

Comment:
Isn't `Infinity` typed as `number` in TS?

Reply:
But does that matter here? 🤔 `assert.isFinite` throws if the given `value` is either not a number, or is a number but not finite. So after calling `assert.isFinite`, `value` is guaranteed to be a `number`. But I could be misunderstanding your concern? 🙇‍♂️

Reply:
Sorry, I was thinking of something else.  You're correct that there's nothing interesting about the `Infinity` case.